.yarnrc.yml

```diff
@@ -1,3 +1,7 @@
----
```

Comment:
dont use yarn much - to be honest not sure what this is doing, i ran the docs locally to make sure some of the links worked and it updated the project to yarn 4.1.1, going to check with the dev site team to make sure this is okay

Reply:
when i installed yarn with node 18, i accidentally updated our docs to yarn 4.1.1, waiting for the dev site team to confirm this update is okay, if it is i'll update the README instructions too (replace node 16 with 18, yarn 4.1.1, etc.)

Reply:
dev site team says 4.1.1 should be okay so i'll update the README